Shamaz/design.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  design.swift
+//  Shamaz
+//
+//  Created by Alexandra Ivanova on 01/04/2020.
+//  Copyright © 2020 Alex Ivanova. All rights reserved.
+//
+
+import UIKit
+
+struct Design {
+    let textColor = UIColor(white: 1.00, alpha: 1.00)
+    let labelFont = UIFont(name: "Chalkduster", size: 30.0)
+    let buttonFont = UIFont(name: "Chalkduster", size: 20.0)
+    let buttonRadius:CGFloat = 5.00
+    let lightGreenColor = UIColor(red:0.56, green:1.00, blue:0.66, alpha:1.00)
+    let lightBlueColor = UIColor(red:0.45, green:1.00, blue:0.97, alpha:1.00)
+    let midnightBlue = UIColor(red:0.13, green:0.16, blue:0.22, alpha:1.00)
```

Comment:
Pro tip: In big applications, using theses names can create issues. I have 2 scenarios in mind:
1. Rebranding of the app, suddenly, you have to replace `lightGreenColor` by `pinkColor` everywhere in your app.
2. Dark mode, for dark mode user `lightGreenColor` should actually display `darkGreenColor`.